### ISSUE TITLE
Coaching Staff: 5.5 Fouls, 8.2 Team Structure

### ISSUE THREAD

**Issue by JoshJetto** [gryphonshafer/Quizzing-Rule-Book#184]
I propose amending the language in 5.5 Fouls and 8.2 Team Structure that would allow for more than one assistant coach.
The current language in Rule 5.5 includes as a foul-able infraction: "Having more than the designated team members, 1 coach, and 1 assistant coach on the quiz platform during a timeout."
My proposed change would be: "Having more than the designated team members and registered coaching staff on the quiz platform during a timeout." 
Rationale: Although most often a team will only have 1 coach and 1 assistant coach at Internationals, there may be scenarios in which a District decides to send more than two coaches for a team. Case-in-point: This summer North Central District is sending 4 coaches for the purpose of equipping some younger/new assistant coaches to coach at the International level. By only allowing 1 coach and 1 assistant coach on the platform during a timeout, this limits the exposure of these younger/newer coaches to live coaching during timeouts at Internationals. In other words, I as the head coach, will only be able to bring one of our assistants up during timeouts, meaning that the financial cost being expended by our District will not result in these assistant coaches gaining as much experience coaching for the money we are spending to equip them. I do not believe that having more than 2 coaches on the platform advantages a team that has only 2 there, so I don't see a need to strictly limit the number that can be on the platform, provided they are registered for Internationals as coaches and not as spectators.
Changing this rule would also necessitate changing rule 8.2 which states: "Each team must have a coach and may also have an assistant coach." I would propose amending this to state: "Each team must have a coach and may also have one or more assistant coaches."

**Comment by jttower** [comment]
I am interested in this change as well, since one year I did bring 2 assistant coaches to IBQ (though I was only allowed to register one).  We definitely need to be training up other coaches, and changing this rule would facilitate that.  However, I could also see an argument being made that things can get crowded on the platform, so maybe there should be a limit of 1 coach and 2 assistant coaches on the platform.

**Comment by levikoral** [comment]
Two assistant coaches seems over-excessive; and I thought that assistant coaches were allowed on the quizzing platform. If not, that should most definitely be changed.

**Comment by josiah-leinbach** [comment]
I agree there is no need to impose a limit of one assistant coach, although the logistics of multiple people being on the platform would be the main challenge. @jttower's proposal of imposing a limit of coaches on the platform would be a good comprise around that. Teams already have multiple _de facto_ extra coaches anyway in the audience who can share information, insights, etc.
